atomata/capstone_WebClient
# Experience Preview V2

### Summary 
Added ability to preview experience again
### Details
- Keyboard inputs now move between actions
- Up/Right to go forward
- Down/Left to go back
- Escape to exit preview
### How to test?
1. Log in
2. Have an experience with multiple actions and text
3. Click the play button on the button of the sidebar (above the save button)
4. Go through each action and ensure all text matches
5. Swap actions and ensure the results are the same
### Checks
